src/utils/deprecationWarning.js

```diff
@@ -12,3 +14,14 @@ export default function deprecationWarning(oldname, newname, link) {
     }
   }
 }
+
+export function collapsable(props, propName, componentName) {
+  if (props[propName] !== undefined) {
+    deprecationWarning(
+      `${propName} in ${componentName}`,
+      'collapsible',
+      'https://github.com/react-bootstrap/react-bootstrap/issues/425'
+    );
+  }
+  return React.PropTypes.bool.call(null, props, propName, componentName);
+}
```

Comment:
I'd rather not see deprecation specific warnings here as it will cause this file to churn a lot. How about putting this in it's own file? Then when it's time to remove this old logic it's as simple as deleting some files with little footprint of code files that will be retained.